src/pages/index.tsx

```diff
@@ -146,17 +147,25 @@ const Home: NextPage<any> = () => {
     utils.user.requests.me.invalidate();
   };
   const connectEmail = async (email: string | null) => {
-    const msg = {
-      to: email, // Replace with your recipient
-      from: "devashishsood9@gmail.com", // Replace with your verified sender
-      subject: "New Contact Message",
-      text: `Dior Dior`,
-    };
+    if (user && email) {
+      const msg: emailSchema = {
+        destination: email,
+        subject: "Carpool Connect Request",
+        body: user.bio,
```

Comment:
Is the `user.bio` the current user's bio from the DB, or is it the text inputted from the ConnectModal component? 

I am asking because in the case where a user edits their bio before they send the request, we should be pulling the text from ConnectModal `textarea` tag.

Reply:
It isn't updated currently. I tried doing that and it turned into a lot of refactoring which was outside the scope of this ticket. Will probably need to have another ticket for that.

Reply:
Overall the Sidebar -> Modal -> Function flow we have for a bunch of stuff will have to be fixed, which is when we can fix this too